projects/GKv3/GEDKeeper3/GKProgram.cs

```diff
@@ -66,14 +66,10 @@ public static void Main(string[] args)
             using (var tracker = new SingleInstanceTracker(GKData.APP_TITLE, AppHost.GetSingleInstanceEnforcer)) {
                 if (tracker.IsFirstInstance) {
                     AppHost.InitSettings();
-                    try {
-                        var appHost = (EtoAppHost)AppHost.Instance;
-                        appHost.Init(args, false);
+                    var appHost = (EtoAppHost)AppHost.Instance;
+                    appHost.Init(args, false);
 
-                        application.Run();
-                    } finally {
-                        AppHost.DoneSettings();

```

Comment:
This finally is never reached on Mac for some reason. I suspect that Mac forcible shuts runtime when `Application.Quit()` is called.